---
title: FailFastException
ms.date: 03/30/2017
ms.assetid: f30de0d9-3a91-4800-9d09-3504d7e5a10f
ms.openlocfilehash: 01c666e816ee39e471b4f6a9728ebdb84941b0d9
ms.sourcegitcommit: bc293b14af795e0e999e3304dd40c0222cf2ffe4
ms.translationtype: MT
ms.contentlocale: es-ES
ms.lasthandoff: 11/26/2020
ms.locfileid: "96284308"
---
# <a name="failfastexception"></a>FailFastException

Id: 103  
  
 Gravedad: error  
  
 Categoría: seguimiento  
  
## <a name="description"></a>Descripción  

 Se inició una excepción durante FailFast. El evento enumera la excepción, el nombre del proceso y el id. del mismo.  
  
## <a name="see-also"></a>Vea también

- [Registro de eventos](index.md)
- [Referencia general de eventos](events-general-reference.md)
